Effects/LCAS.hlsl

```diff
@@ -0,0 +1,64 @@
+// This is a combination of linear interpolation and light version of cas
+
+//!MAGPIE EFFECT
+//!VERSION 1
+
+//!CONSTANT
+//!VALUE INPUT_PT_X
+float inputPtX;
+
+//!CONSTANT
+//!VALUE INPUT_PT_Y
+float inputPtY;
+
+//!TEXTURE
+Texture2D INPUT;
+
+//!SAMPLER
+//!FILTER LINEAR

```

Comment:
A point sampler is sufficient to fetch the neighborhood pixels

Reply:
added sharpening parameter sir. will try point sampler

Reply:
The visual degraded significantly sir, maybe due to linear to nearest.

Reply:
They should have the same effect since there's no bilinear interpolation here. I will test it later.